CHANGELOG.rst

```diff
@@ -16,6 +16,11 @@ Unreleased
 ~~~~~~~~~~
 
 
+[3.7.2] - 2020-08-10
+~~~~~~~~~~~~~~~~~~~~
+* plugins.__init__.py was missing some objects used by users, this release adds:
+    - PluginSignals, PluginContexts, PluginError, get_url
```

Comment:
This needs to be updated.  I think `get_url` shouldn't be here.  I would drop this list, and just add "See `__init__.py` for details."  
Also, I'd mention that the name changes are backward incompatible changes, but that this is being put out as a bug fix because this is the initial releases that isn't yet in use.